hack/kivik-set-zero.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/vendor/github.com/go-kivik/kivik/iterator.go b/vendor/github.com/go-kivik/kivik/iterator.go
+index f12bf985..4bc7f1cb 100644
+--- a/vendor/github.com/go-kivik/kivik/iterator.go
++++ b/vendor/github.com/go-kivik/kivik/iterator.go
+@@ -79,6 +79,8 @@ func (i *iter) next() (doClose, ok bool) {
```

Comment:
:see_no_evil:   any idea when they fix ? or release this code ? :grinning:

Reply:
no idea. I'm watching :-)